src/module.ts

```diff
@@ -47,9 +47,11 @@ export default defineNuxtModule<ModuleOptions>({
       getContents: () => `export default ${JSON.stringify(options.rules, null, 2)}`
     }).dst || '')
 
-    nuxt.hook('nitro:build:before', (nitro) => {
-      nitro.options.prerender.routes.push(`/${ROBOTS_FILENAME}`)
-    })
+    if (nuxt.options.target === 'static') {
```

Comment:
Use `if (nuxt.options._generate) {` instead of `if (nuxt.options.target === 'static') {`

The user does not always define the target

Reply:
I updated the test if you don't mind 😄

Reply:
Ahh, `nuxtConfig` it is! Thank you :heart: it's kind of hard to find documentation for the edge test utils, right? Or have I just looked at wrong places?

Reply:
The documentation is very simple https://v3.nuxtjs.org/getting-started/testing

I know the options because I helped to develop 😅